src/main/features/claim/views/start.njk

```diff
@@ -10,67 +10,76 @@
   <div class="grid-row">
     <div class="column-two-thirds">
       <h1 class="heading-large">{{ t('Make a money claim') }}</h1>
+      <br>
```

Comment:
don't use BR! use css to get desired effect .form-group .form-group-related are useful classes to add

Reply:
@robertparkinson Can you please elaborate